src/sa_web/static/js/routes.js

```diff
@@ -7,6 +7,11 @@ var Shareabouts = Shareabouts || {};
     routes: {
       '': 'viewMap',
       'place/new': 'newPlace',
+      'place/new/observation': 'newPlaceObservation',
+      'place/new/question': 'newPlaceQuestion',
+      'place/new/idea': 'newPlaceIdea',
+      'place/new/complaint': 'newPlaceComplaint',
+      'place/new/greenwall': 'newPlaceGreenwall',
```

Comment:
Good idea. Can we specify routes in the flavor config file? And then have them loaded into that override routes.js file. This way, in the future people who build or maintain their own flavors only have to edit one file to add or edit a new form or content page.